pages/services/data-science-engine/1.0.0/install/index.md

```diff
@@ -0,0 +1,402 @@
+---
+layout: layout.pug
+navigationTitle: Install and Customize
+excerpt: Customize your instance of DC/OS Data Science Engine
+title: Install
+menuWeight: 3
+model: /services/data-science-engine/data.yml
+render: mustache
+---
+This section provides detailed instructions for installing and customizing your {{ model.techName }} deployment. If you just want to get a cluster up and running for testing or evaluation, see the [Quick Start Guide](/services/data-science-engine/1.0.0/quick-start/) for instructions.
+
+# Prerequisites
+
+- DC/OS and DC/OS CLI installed
+- Depending on your security mode], {{ model.packageName }} requires service authentication for access to DC/OS.
+
+| Security Mode | Service Account |
+|----------------|----------------|
+| Disabled | Not available |
+| Permissive | Optional |
+| Strict | Required |
+
+For more information about service accounts, see the [Security](/services/data-science-engine/1.0.0/security/) documentation. 
+
+# Default installation
+
+## From the CLI
+
+To install the {{ model.techName }} service from the DC/OS CLI, run the following command:
+
+```bash
+dcos package install  {{ model.packageName }}
```

Comment:
There is no CLI, right @akirillov ?

Reply:
This refers to the DC/OS CLI, which certainly exists. Do you mean that Data Science Engine cannot be installed from the DC/OS CLI, or that it does not include a set of its own CLI commands?

Reply:
sorry about that, I meant [this section](https://github.com/mesosphere/dcos-docs-site/pull/2440/files/20cfde8766eb71718ee0bd3de92783c72ad8e04c#diff-ffe1d5ef0fc70685bc6e4defb4b0012eR35-R43).

There is no data-science-engine CLI component

Reply:
Got it, thanks. Deleted highlighted section.